l5-operator/controllers/bestie_controller.go

```diff
@@ -76,37 +74,13 @@ func (r *BestieReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 
 	var result *reconcile.Result
 
-	result, err = r.ensureSecret(ctx, bestie, r.mysqlAuthSecret(bestie))
```

Comment:
we should still do ensure for the postgres secret

Reply:
True. I'll create a placeholder for now.